src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  margin: 0;
+}
+
 body {
   margin: 0;
 }
```

Comment:
You've applied 'margin: 0;' to both the 'html' and 'body' selectors. This is not necessary. You can remove the margin reset from the 'body' selector as the 'html' selector will cover the entire document.